docs/api/elevation/api-reference.md

```diff
@@ -14,10 +14,6 @@ An elevation service request takes the form of `servername/height?json={}`, wher
 
 There is an option to name your elevation request. You can do this by appending the following to your request `&id=`.  The `id` is returned with the response so a user could match to the corresponding request.
 
-## Using the hosted Mapbox Elevation Service
```

Comment:
imho removing it seems ok. its only purpose was to let people know that a hosted service existed, and since it doesnt no point in mentioning? i suppose mentioning that it did would just be for historical purposes?

Reply:
Yeah, let's just have these docs be about the Valhalla software itself. Any hosted deployments would probably need their own additional docs (to describe API keys and whatnot)

Reply:
Cool.